search-commons/src/main/java/no/unit/nva/search2/resource/ResourceQuery.java

```diff
@@ -151,6 +155,11 @@ protected boolean isPagingValue(ResourceParameter key) {
         return key.ordinal() >= FIELDS.ordinal() && key.ordinal() <= SORT_ORDER.ordinal();
     }
 
+    @Override
+    protected Map<String, String> aggregationsDef() {
```

Comment:
Why not to name with full name? 
`aggregationDefinitons()`
All these abbreviations does not make a lot of sense for one who does not know the context